mirascope/wandb/prompt.py

```diff
@@ -105,10 +156,11 @@ def trace(
             inputs={message["role"]: message["content"] for message in self.messages},
```

Comment:
on lines 154/155 we're using the start/end times in dump. Instead we should use self._start_time and self._end_time. Same is true for `_trace_error`

Reply:
since create catches exceptions too now, going to delete the `WandbPrompt._creation_time_ms` attr as well since no longer necessary, same with dump